### ISSUE TITLE
Cannot connect to game.

### ISSUE THREAD

**Issue by R3Skako** [KaosDrip/Xensis#27]
When with the bot i get this error: 
![obraz](https://github.com/KaosDrip/Xensis/assets/80347705/7ec45fce-1ebc-41ca-99b5-e9375a7da69a)

**Comment by Aspect2222** [comment]
> When with the bot i get this error: ![obraz](https://private-user-images.githubusercontent.com/80347705/272038786-7ec45fce-1ebc-41ca-99b5-e9375a7da69a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIxMTY0MDAsIm5iZiI6MTcwMjExNjEwMCwicGF0aCI6Ii84MDM0NzcwNS8yNzIwMzg3ODYtN2VjNDVmY2UtMWViYy00MWNhLTk5YjUtZTkzNzVhN2RhNjlhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA5VDEwMDE0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEwMmVkYzA5OTY5MGFiMDQ2MGNkZjVlOGY2MTY5ODliZjdkYTAzOTBiY2YzNzc1M2JlYTg1YTNjN2YyY2RhZTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.pCfh8V7Yc-jnUdh2GTv-1jlt9yiQJwmXLo2avBW1kLQ)

Hi there, please download newest version here from github and let me know if it works now. Thank You

**Comment by Tiger15cub** [comment]
hi, im on latest version, and i am getting the party failed to connect error.

**Comment by Aspect2222** [comment]
> hi, im on latest version, and i am getting the party failed to connect error.

i am able to join the bot but skins and more not working, i am working on a fix

**Comment by Tiger15cub** [comment]
> > hi, im on latest version, and i am getting the party failed to connect error.
> 
> i am able to join the bot but skins and more not working, i am working on a fix

yeah, i can join the bot but skins do not work and matchmaking does not work.

**Comment by badaact** [comment]
hi @Aspect2222 did you find any solution for the issue that skins are invisble?

**Comment by Aspect2222** [comment]
> hi [@Aspect2222](https://github.com/Aspect2222) did you find any solution for the issue that skins are invisble?

Hello, FortnitePY was discontinued. We will switch to RebootPY now. Will update it later today/tomorrow after years.

**Comment by Aspect2222** [comment]
> hi [@Aspect2222](https://github.com/Aspect2222) did you find any solution for the issue that skins are invisble?

fixed
